recipe/build.sh

```diff
@@ -18,12 +25,20 @@ do
 	    --disable-overwrite \
 	    --enable-symlinks \
 	    --enable-termcap \
-	    --with-termlib \
 	    $WIDEC_OPT \
 	    --with-terminfo-dirs=/usr/share/terminfo
     make
     make install
 
+    # Provide libtinfo as a link to libncurses
+    LIB_LETTER=""
+    if [ "${USE_WIDEC}" = true ];
+    then
+        LIB_LETTER="w"
+    fi
+    ln -s "${PREFIX}/lib/libncurses${LIB_LETTER}.a" "${PREFIX}/lib/libtinfo${LIB_LETTER}.a"
+    ln -s "${PREFIX}/lib/libncurses${LIB_LETTER}.${DYLIB_EXT}" "${PREFIX}/lib/libtinfo${LIB_LETTER}.${DYLIB_EXT}"
```

Comment:
Just symlink the libraries. This will ensure anything that tries to link against `libtinfo` (which will no longer exist after this addition) will pull their symbols from `libncurses` (where they all are) instead of having an error.